ci/Jenkinsfile.combined

```diff
@@ -80,11 +80,39 @@ pipeline {
             }
           }
         }
-        stage('Windows/x86_64') { steps { script {
-          windows_x86_64 = getArtifacts(
-            'Windows', jenkins.Build('status-desktop/systems/windows/x86_64/package')
-          )
-        } } }
+        stage('Windows/x86_64 and E2E') {
+          stages {
+            stage('Windows/x86_64') {
+              steps {
+                script {
+                  windows_x86_64 = getArtifacts(
+                    'Windows', jenkins.Build('status-desktop/systems/windows/x86_64/package')
+                  )
+                }
+              }
+            }
+            stage('E2E') {
+              when { 
+                expression { 
+                  env.JOB_NAME.toLowerCase().contains('nightly') 
+                } 
+              }
```

Comment:
If this is conditional, why not make the whole `Windows/x86_64 and E2E` stage conditional on that?
The step before it is pointless if we don't run the tests.

Reply:
not now , maybe lets these tests run for a while as non mandatory to pass. I think they would work fine, but who knows and restarting release job is pain